Example/Podfile

```diff
@@ -1,8 +1,10 @@
+platform :ios, '9.0'
 source 'https://github.com/CocoaPods/Specs.git'
 use_frameworks!
 
 target 'CMHealth_Example' do
   pod 'CMHealth', :path => '../'
+  #pod "CareKit", "~> 1.0"
```

Comment:
Yes, since CareKit is now a dependency of CMHealth it is pulled into the lock file. Should probably have cleaned up this comment. Will make sure to do that next time I touch the Podfile.